plan.go

```diff
@@ -2837,17 +2839,20 @@ func (r *emptyFieldsPlan) do(ctx *execCtx, f func(id interface{}, data []interfa
 
 type wrapFilterPlan struct {
 	*filterDefaultPlan
+	once  sync.Once
```

Comment:
> Thanks for the feedback. I will run with -race just to be sure.

For that to be really effective, please add a test that exercises concurrency for a query of this type ;-)